src/extension/netcdf-out/src/main/resources/GeoServerApplication.properties

```diff
@@ -1,6 +1,7 @@
 NetCDFOut.attribute = Attribute
 NetCDFOut.compressionLevel = NetCDF-4 Compression Level (0-9, 0 = UNCOMPRESSED)
-NetCDFOut.copyAttributes = Copy Attributes from NetCDF/GRIB Source
+NetCDFOut.copyVariableAttributes = Copy Variable Attributes from NetCDF/GRIB Source
```

Comment:
This is the right place to add the "Variable", as you have done. The property name can stay the same but the text you can fix.